### ISSUE TITLE
[APIView Python] inserts additional text if paramtype is used for :keyword arguments

### ISSUE THREAD

**Issue by LarryOsterman** [Azure/azure-sdk-tools#1619]
If you have a type which uses :paramtype: for keyword parameters, 

```
class AttestationClient(object):
    """An AttestationClient object enables access to the Attestation family of APIs provided
      by the attestation service.

    :param str instance_url: base url of the service
    :param credential: Credentials for the caller used to interact with the service.
    :type credential: :class:`~azure.core.credentials_async.AsyncTokenCredential`
    :keyword pipeline: If omitted, the standard pipeline is used.
    :paramtype pipeline:  ~AsyncPipelineClient
    :keyword transport: If omitted, the standard pipeline is used.
    :paramtype transport: AsyncHttpTransport 
    :keyword policies: If omitted, the standard pipeline is used.
    :paramtype policies: list[AsyncHTTPPolicy]

    For additional client creation configuration options, please see https://aka.ms/azsdk/python/options.

    """
```
the generated code has some bogus text inserted:

def __init__(
--
+ | self,
+ | credential: `AsyncTokenCredential`,
+ | instance_url: str,
+ | *,
+ | pipeline: AsyncPipelineClient = ...,
+ | policies: list[AsyncHTTPPolicy]    For additional client creation configuration options, please see http = ...,
+ | transport: AsyncHttpTransport = ...,
+ | **kwargs
+ | ) -&gt; None
&nbsp; | &nbsp;

Note the inserted "For additional client creation configuration" text - that was caused by the :paramtype parser.

**Comment by kurtzeborn** [comment]
@seankane-msft may be more appropriate to work on this one.

**Comment by tjprescott** [comment]
I believe this has been fixed. I could not reproduce it:

![image](https://user-images.githubusercontent.com/5723682/141013619-03e11e98-5ddd-4073-a47b-b7a6da2e7199.png)

![image](https://user-images.githubusercontent.com/5723682/141013675-a98ff33c-3dbb-4792-832d-17b5c72e80e7.png)

**Comment by tjprescott** [comment]
This occurs due to the regex used to capture definitions that flow onto multiple lines.

**Comment by tjprescott** [comment]
I believe this is fixed in APIView 0.2.6. Please reopen if not the case.
